test/browser-plugins/plugins.test.ts

```diff
@@ -364,7 +364,9 @@ describe('Plugins', () => {
                     browser = await plugin.launch(launchContext);
                     const contexts = browser.contexts();
                     expect(contexts).toHaveLength(1);
-                    expect(contexts[0]).toEqual((browser as Browser).browserContext);
+                    // Cast to any to access private property
+                    // eslint-disable-next-line no-underscore-dangle
+                    expect(contexts[0]).toEqual((browser as any)._browserContext);
```

Comment:
No need to cast, use square bracket accessing

Reply:
`_browserContext` does not exist on `PlaywrightBrowser`, wdym?

Reply:
How does it not exist if you test for it? 👀

Reply:
`Browser` (`PlaywrightBrowserWithPersistentContext`) is cast to `PlaywrightBrowser` which doesn't have this property. We could cast `browser` to `PlaywrightBrowserWithPersistentContext` back but it's just faster to cast it to `any`.